libclc/CMakeLists.txt

```diff
@@ -378,6 +379,18 @@ foreach( t ${LIBCLC_TARGETS_TO_BUILD} )
                 # have to be explicity declared in the soruce.
                 list( APPEND flags -Xclang -fdeclare-spirv-builtins)
 
+                # OpenCL 3.0 extensions
+                list( APPEND flags -cl-std=CL3.0 "SHELL:-Xclang" )
+                string(CONCAT CL_3_0_EXTENSIONS
+                  "-cl-ext="
+                  "+cl_khr_fp16,"
+                  "+cl_khr_fp64,"
+                  "+__opencl_c_3d_image_writes,"
```

Comment:
Why do we need image related extensions in this PR?

Reply:
This is part of the generic bump libclc version to 3.0 PR. Will rebase